.github/workflows/actions.yml

```diff
@@ -62,6 +62,16 @@ jobs:
           flags: keras.applications,keras.applications-${{ matrix.backend }}
           files: apps-coverage.xml
           fail_ci_if_error: true
+      - name: Test integrations
+        run: |
+          python integration_tests/distribute_training_test.py
+          python integration_tests/import_test.py
+          python integration_tests/numerical_test.py
+      - name: Test backend-specific integrations
+        if: ${{ matrix.backend == 'torch'}}
+        run: |
+          python integration_tests/layer_in_torch_workflow.py
+          python integration_tests/torch_backend_keras_workflow.py
```

Comment:
What this one is testing is unclear. Doesn't seem to match the title

Reply:
I've deleted this integration test, as it's not relevant. It's workflow is tested effectively elsewhere throughout Keras.